package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "radonjs",
+  "version": "1.0.0",
+  "description": "Radon is an object-oriented state management framework for JavaScript applications.",
+  "main": "stateNode.js",
+  "scripts": {
+    "test": "jest --watch"
```

Comment:
Make Different Test for Each File, so the team doesn't need to run all the test just to test their section.